task/upload.go

```diff
@@ -47,6 +47,7 @@ var (
 	OutputNameOSSourceMP4   = OutputName("source_mp4")
 	OutputNameOSPlaylistHLS = OutputName("playlist_hls")
 	OutputNameIPFSSourceMP4 = OutputName("ipfs_source_mp4")
+	OutputNameClipSource    = OutputName("clip_source")
```

Comment:
Are clip outputs different from regular MP4/HLS? I thought it was everything the same, only changing the input. We could re-use a lot of logic if they were the same, like on the API to process the task output.

Reply:
Just to clarify, the step 3 is provided like a "source playback" workflow right? Or would the job complete from the `task-runner` side but stay running on Catalyst?

Reply:
Exactly like a source playback for assets!

Reply:
Exactly like a source playback for assets!